tools/topology2/get_abi.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2019 Intel Corporation. All rights reserved.
```

Comment:
Please add `set -e`.

Please move all the code to a `main "$@"` function, see example in `build-tools.sh` above.
Pros:
- can declare other functions in any order
- can use `local` variables to avoid shadowing
- easier to `source` for interactive debug, just comment out the last line
- keeps the same indentation level and minimizes diff when moving code around functions

Cons:
- None